src/components/NewTableComponent/PenaltyModal.jsx

```diff
@@ -29,12 +29,9 @@ const PenaltyModal = ({ isOpen, onClose, studentId }) => {
         ];
 
         try {
-            const response = await fetch(`${baseUrl}/api/v1/penalties`, {
+            const response = await apiCall('/penalties', {
```

Comment:
let us make a function for this in api.ts and then use here, similar to `fetchEachJob`

Reply:
do this for all the placess